about/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <h1>Intellect Fox</h1>
+        <h3>Front-end 4</h3>
+        <hr/>
+        <h5>Почему я выбрала направление Front-end разработки &#10067</h5>
```

Comment:
Tag `head` shouldn't include `h1, h3, hr, h5` tags.  That tag only for metadata for the browser. Please move that to `body` tag.

Reply:
Moved to body tag.